src/control/server/storage/scm/ipmctl.go

```diff
@@ -180,8 +178,16 @@ func (cr *cmdRunner) deleteGoals() error {
 	return errors.Wrap(cr.binding.DeleteConfigGoals(cr.log), "failed to delete config goals")
 }
 
-func (cr *cmdRunner) createNamespace() (string, error) {
-	return cr.runCmd(cmdCreateNamespace)
+func (cr *cmdRunner) createNamespace(regionIdx int, sizeBytes uint64) (string, error) {
+	cmd := cmdCreateNamespace
+	if regionIdx > -1 && regionIdx < 2 {
```

Comment:
nit: These magic values should probably be consts. Is `-1` even a valid value? If not, maybe `regionIdx` should be a uint.